packages/state-transition/src/util/attesterStatus.ts

```diff
@@ -19,27 +19,6 @@ const TIMELY_SOURCE = 1 << TIMELY_SOURCE_FLAG_INDEX;
 const TIMELY_TARGET = 1 << TIMELY_TARGET_FLAG_INDEX;
 const TIMELY_HEAD = 1 << TIMELY_HEAD_FLAG_INDEX;
 
-/**
```

Comment:
since we remove `AttesterStatus`, we may remove this file completely or remove most of the functions, need to remove all consumers of removed functions in this file too